src/Proxies/WP.php

```diff
@@ -177,4 +178,114 @@ public function number_format_i18n( float $number, int $decimals = 0 ): string {
 	public function wp_doing_ajax(): bool {
 		return wp_doing_ajax();
 	}
+
+	/**
+	 * Retrieves an array of the latest posts, or posts matching the given criteria.
+	 *
+	 * @since x.x.x
+	 *
+	 * @see WP_Query
+	 * @see WP_Query::parse_query()
+	 *
+	 * @param array $args {
+	 *     Arguments to retrieve posts. See WP_Query::parse_query() for all available arguments.
+	 * }
+	 * @return WP_Post[]|int[] Array of post objects or post IDs.
+	 */
+	public function get_posts( array $args ): array {
+		return get_posts( $args );
+	}
+
+	/**
+	 * Gets a list of all registered post type objects.
+	 *
+	 * @since x.x.x
+	 *
+	 * @param array|string $args     Optional. An array of key => value arguments to match against
```

Comment:
In the PHPDOCS it says `array|string` for `$args` param. However, in the method signature it claims only `array` for `$args` attribute.

Reply:
I remove the `array` type as union types are only allowed in PHP >=8.0